VideoApi/VideoApi/Controllers/ConferenceController.cs

```diff
@@ -478,6 +480,44 @@ public async Task<IActionResult> GetHostsInHearingsTodayAsync()
 
             return Ok(response);
         }
+        
+        /// <summary>
+        /// Get conferences Hearing rooms
+        /// </summary>
+        /// <returns>Hearing rooms details</returns>
+        [AllowAnonymous]
+        [HttpGet("dateStamp/hearingRooms")]
```

Comment:
dateStamp doesn't really mean anything here does it? Is there a more descriptive name we could use? Or perhaps use parameter binding so the date is part of the url rather than the query.

Reply:
fixed. thanks